src/payment/checkout/payment-form/FormInput.jsx

```diff
@@ -19,7 +22,7 @@ const FormInput = ({
       id={id}
       disabled={disabled}
     />
-    {touched && error && <span className="text-danger">{error}</span>}
+    {touched && error && <span className="text-danger">{intl.formatMessage(messages[error])}</span>}
```

Comment:
Sanity checking - Is it the case that all error messages are coming back as message keys now, instead of formatted messages?  I see code in here elsewhere where we're definitely doing that, but I can't recall if there are other field errors that may find their way in here that were pre-formatted.

Reply:
I believe that I got everything, but I'm not sure how I'd verify that 100%. I could add code to check whether `error` is in `messages`, and if it isn't just display `error` directly.